### ISSUE TITLE
Could cmder make Universal Shortcut for all shell ?

### ISSUE THREAD

**Issue by eromoe** [cmderdev/cmder#1218]
I have posted a issue about shortcut before https://github.com/cmderdev/cmder/issues/1080 , sadly with no reply.

Even offcial site http://cmder.net/ shortcut not work on git-bash (though `ctrl` +`w` or somethin else is bash shortcut),  I think we should make a Universal Shortcut work on all shell.

**Comment by Jackbennett** [comment]
Shortcut to do what? [`ctrl + w` to delete the line?](https://www.ice2o.com/bash_quick_ref.html)

It's difficult to understand what you want cmder to do.

The powershell task ships PSReadLine so you have;
keys | function | description
-|-|-
Ctrl+End       |       ForwardDeleteLine         |    Delete text from the cursor to the end of the line
Ctrl+Home       |      BackwardDeleteLine       |     Delete text from the cursor to the start of the line

see the rest with `Get-PSReadlineKeyHandler`

**Comment by eromoe** [comment]
I mean use Universal(Global Uinque) Shortcut of actions for all shell.  As you refer link, `ctrl + w` is use to delete line in git-bash, but not work in cmd or powershell. I think if there a mapping would be much better. 
For example, 
1.  if I set key mapping to powershell-style,  `Ctrl+End`  delete line in git-bash.
2. if I set key mapping to bash-style, ,  `Ctrl+ w` would delete line in powershell/cmd.



people who like powershell  choose the powershell key mapping,  bash
